tfhe-benchmark/benches/core_crypto/ks_bench.rs

```diff
@@ -423,120 +423,148 @@ mod cuda {
+                        bit_size,
+                        vec![bit_size],
+                    );
                 }
                 BenchmarkType::Throughput => {
                     let gpu_keys_vec = cuda_local_keys_core(&cpu_keys, None);
                     let gpu_count = get_number_of_gpus() as usize;
 
-                    bench_id = format!("{bench_name}::throughput::{name}");
-                    let blocks: usize = 1;
-                    let elements = throughput_num_threads(blocks, 1);
-                    let elements_per_stream = elements as usize / gpu_count;
-                    bench_group.throughput(Throughput::Elements(elements));
-                    bench_group.sample_size(50);
-                    bench_group.bench_function(&bench_id, |b| {
-                        let setup_encrypted_values = || {
-                            let local_streams = cuda_local_streams_core();
-
-                            let plaintext_list = PlaintextList::new(
-                                Scalar::ZERO,
-                                PlaintextCount(elements_per_stream),
-                            );
-
-                            let input_cts = (0..gpu_count)
-                                .map(|i| {
-                                    let mut input_ct_list = LweCiphertextList::new(
-                                        Scalar::ZERO,
-                                        big_lwe_sk.lwe_dimension().to_lwe_size(),
-                                        LweCiphertextCount(elements_per_stream),
-                                        params.ciphertext_modulus.unwrap(),
-                                    );
-                                    encrypt_lwe_ciphertext_list(
-                                        &big_lwe_sk,
-                                        &mut input_ct_list,
-                                        &plaintext_list,
-                                        params.lwe_noise_distribution.unwrap(),
-                                        &mut encryption_generator,
-                                    );
-                                    let input_ks_list = LweCiphertextList::from_container(
-                                        input_ct_list.into_container(),
-                                        big_lwe_sk.lwe_dimension().to_lwe_size(),
-                                        params.ciphertext_modulus.unwrap(),
-                                    );
-                                    CudaLweCiphertextList::from_lwe_ciphertext_list(
-                                        &input_ks_list,
-                                        &local_streams[i],
-                                    )
-                                })
-                                .collect::<Vec<_>>();
+                    for uses_simple_indices in [false, true] {
+                        bench_id = format!(
+                            "{bench_name}::throughput::simple_{uses_simple_indices}::{name}"
```

Comment:
In the bench name you can specify whether it's gemm ks or classical keyswitch that's benched